src/api/app/models/issue_tracker.rb

```diff
@@ -32,6 +33,14 @@ def write_to_backend
     self.issues_updated ||= Time.now
   end
 
+  def verify_name!(name)
```

Comment:
renamed the method like you propose. It might make sense to move it to Issue class, however it won't work without a tracker object.

We can have it also as non-exception class, but we need to move then all source verification code to that style. I don't want to make this single test special now.